### PR TITLE
Thumbnail display

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Use correct variable

# Testing this PR

Test 1:

* as tb1 import a big image via CLI
* in insight as tb2 view the image imported by tb1. A thumbnail should be generated
* Go to web as tb2, check that a thumbnail is displayed

Test 2:

* as tb1 import a big image via insight
* in insight as tb2 view the image imported by tb1. A thumbnail should be generated
* Go to web as tb2, check that a thumbnail is displayed

Test 3:

* as tb1 import a image via CLI using skip thumbnail
* in insight as tb2 view the image imported by tb1. A thumbnail should be generated
* Go to web as tb2, check that a thumbnail is displayed

Test 4:

* as tb1 import a image using insight using skip thumbnail
* in insight as tb2 view the image imported by tb1. A thumbnail should be generated
* Go to web as tb2, check that a thumbnail is displayed

cc @joshmoore @mtbc @rgozim

### CODE REVIEW

File: components/server/src/ome/services/ThumbnailBean.java [hunk 1]
```diff
@@ -1260,16 +1260,22 @@ protected void actOnOneGroup(Set<Long> pixelsIds) {
...
             }
         }
         try {
-            compressThumbnailToDisk(thumbMetaData, image, false);
-            iUpdate.saveObject(thumbMetaData);
+            compressThumbnailToDisk(thumbnailMetadata, image, false);
+            //Saving of thumbnailMetadata already happened in _createThumbnail
+            //if owner != settings but not for owner == settings
+            //This could be moved to _createThumbnail
+            if  (thumbnailMetadata.getDetails().getOwner().getId() ==
```
Comment (mtbc) [inline components/server/src/ome/services/ThumbnailBean.java]: double space :smiley:

Comment (mtbc) [inline components/server/src/ome/services/ThumbnailBean.java]: use `equals` to compare `Long`s

**Comment by mtbc** [comment]
With this PR merged in I am not managing to reproduce any of the bad workflows I found on Friday: functional testing is passing at least for me.

**Comment by jburel** [comment]
@mtbc done

**Comment by mtbc** [comment]
Great, thank you. Initial functional testing still looking good for me with latest commit included.

**Comment by dominikl** [comment]
Tested the same workflows on eel, looks good, didn't notice anything unexpected or errors.

**Comment by joshmoore** [comment]
Merging for inclusion in 5.4.8 (4th times a charm)

**Comment by rgozim** [comment]
Seems better now. Previously, deleting a pyramid and thumbnail resulted in an exclamation mark when user 2 tried to view user 1's image. Now, user 2 sees the correct timer icon.

**Pull request merged**